stackrox/base/persistentvolumeclaims/persistentvolumeclaim.yaml

```diff
@@ -0,0 +1,24 @@
+# Source: stackrox-central-services/templates/01-central-11-pvc.yaml
```

Comment:
Do you for see more PVCs occuring in this NS? Generally if we only need one of a resource we deploy it from the base kustomization, and reference the `base/persistentvolumeclaims/kustomization.yaml` in the `stackrox/base/kustomization.yaml` file.

Reply:
working on